examples/markdown-plugin.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+
+    <title>PyMarkdown</title>
+
+    <link rel="icon" type="image/png" href="favicon.png" />
+    <link rel="stylesheet" href="../build/pyscript.css" />
+
+    <script defer src="../build/pyscript.js"></script>
+
+    <py-config>
+      packages = [
+        "markdown"
+      ]
+      plugins = [
+        "../build/plugins/python/py_markdown.py"
+      ]
+    </py-config>
```

Comment:
I was literally just typing the same thing! At least with regards to the naming. Was also thinking about the possibility of `js-plugins=[...]` at some point, which would presumably need to be treatly slightly differently in their loading.

Reply:
I thought about it but couldn't find a "good" reason why we should create different keys to define the plugins I want to install in my app. Why do we need to add this complexity from a user standpoint?

There's more... imagine a future with a ton of plugins and a plugin index, it may not be clear if a plugin is implemented in `Python`, `JS` or..... something else. I'd not want to confuse users on that and would rather add a process where we are able to know what type of plugin is it that we are downloading (adding the concept of plugin manifest for instance).

Reply:
+1 for having both JS and Python plugins in `config.plugins` *eventually*.

But my doubts are not about JS vs Python, they are about to to interpret the strings in a forward-compatible way. For example, consider this future use cases:
1. `plugins = ["./py_markdown.py"]`: this is your current semantics: fetch a single file from an URL
2. `plugins = ["https://..../py_markdown.py"]`: this works out of the box
3. `plugins = ["py_markdown"]`: fetch the `py-markdown` plugin from the POPI (PyScript Official Plugin Index 😅)
4. `plugins = ["foo", "bar.baz"]`: fetch the plugins `foo` and `bar.baz` from the POPI
5. `plugins = ["foo.py"]`: *ambigous*: is it a relative URL or a plugin called `foo.py` from POPI?

They all seem reasonable, but suddenly point (5) is ambiguous: sure, we could have heuristics to say that if something ends with `.py` is always an URL, or the opposite that something is always a POPI name unless you specify `https://` or `./`, but they look a bit fragile.
But most importantly, we *don't know yet* how we want `config.plugins` to look like in the future.

Basically, I'm just worried that if we use `config.plugins` *now*, it will cause headaches in the future. That's why I wanted to use "something else", and possibly something which makes it clear that it's a preview/alpha/experimental/whatever. So, few suggestions:
```
py_plugins = [...]
_plugins = [...]
experimental_plugins = [...]
plugins_but_dont_use_me_please = [...]   # :)

[experimental]
plugins = [...]

[unstable]
plugins = [...]
```

Reply:
+1 for `experimental.plugins`, so that we are free to completely break it when we move to the main section